packages/tinacms/src/edit-state.tsx

```diff
@@ -38,6 +39,52 @@ export const TinaEditProvider = ({
   )
 }
 
+export function useTina<T extends object>({
```

Comment:
Maybe we could make this default to the type of the data provided.
Something along the lines of
 `T extends object = typeof props.data`

Reply:
It should already infer its type based on the `data` property type
<img width="292" alt="Screen Shot 2022-01-19 at 4 06 09 PM" src="https://user-images.githubusercontent.com/3323181/150205745-fe1f346c-b450-4330-902d-deed93a3e80b.png">

Is that what you mean?

Reply:
Oh perfect 😄